diff --git a/CMakeLists.txt b/CMakeLists.txt
index c2f5bbf..43efcfd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -5,22 +5,22 @@ add_subdirectory("dpf")
 
 ###
 dpf_add_plugin(string-machine
-  TARGETS lv2 vst2
+  TARGETS lv2 #vst2
   UI_TYPE cairo
   FILES_DSP
     "plugins/string-machine/StringMachinePlugin.cpp"
-  FILES_UI
-    "plugins/string-machine/StringMachineUI.cpp"
-    "plugins/string-machine/Artwork.cpp"
-    "sources/ui/components/KnobSkin.cpp"
-    "sources/ui/components/SkinSlider.cpp"
-    "sources/ui/components/SkinToggleButton.cpp"
-    "sources/ui/components/SkinTriggerButton.cpp"
-    "sources/ui/components/SkinIndicator.cpp"
-    "sources/ui/components/PlotView.cpp"
-    "sources/ui/FontEngine.cpp"
-    "sources/ui/Fontstash.cpp"
-    "sources/ui/Cairo++.cpp"
+  #FILES_UI
+  #  "plugins/string-machine/StringMachineUI.cpp"
+  #  "plugins/string-machine/Artwork.cpp"
+  #  "sources/ui/components/KnobSkin.cpp"
+  #  "sources/ui/components/SkinSlider.cpp"
+  #  "sources/ui/components/SkinToggleButton.cpp"
+  #  "sources/ui/components/SkinTriggerButton.cpp"
+  #  "sources/ui/components/SkinIndicator.cpp"
+  #  "sources/ui/components/PlotView.cpp"
+  #  "sources/ui/FontEngine.cpp"
+  #  "sources/ui/Fontstash.cpp"
+  #  "sources/ui/Cairo++.cpp"
   FILES_COMMON
     "plugins/string-machine/StringMachineShared.cpp"
     "plugins/string-machine/StringMachinePresets.cpp"
@@ -64,7 +64,7 @@ endif()
 
 ###
 dpf_add_plugin(string-machine-chorus
-  TARGETS lv2 vst2
+  TARGETS lv2 #vst2
   FILES_DSP
     "plugins/string-machine-chorus/ChorusPlugin.cpp"
   FILES_COMMON
@@ -91,7 +91,7 @@ endif()
 
 ###
 dpf_add_plugin(string-machine-chorus-stereo
-  TARGETS lv2 vst2
+  TARGETS lv2 #vst2
   FILES_DSP
     "plugins/string-machine-chorus-stereo/StereoChorusPlugin.cpp"
   FILES_COMMON
diff --git a/dpf b/dpf
--- a/dpf
+++ b/dpf
@@ -1 +1 @@
-Subproject commit f24e5317cd7b57c8ab36b0987ec29a5579cd5dd2
+Subproject commit f24e5317cd7b57c8ab36b0987ec29a5579cd5dd2-dirty
